policyengine-client/src/common/advanced.jsx

```diff
@@ -1,15 +1,17 @@
 import { LoadingOutlined } from '@ant-design/icons';
-import { Collapse, Spin, Alert, Table, Tooltip } from 'antd';
+import { Collapse, Spin, Alert, Table, Tooltip, Switch } from 'antd';
```

Comment:
I can't find the element this is called, but I think this would be clearer, since there's no natural True/False for absolute/relative change:
![image](https://user-images.githubusercontent.com/6076111/141505623-3a735e01-3382-431b-9b27-ea42e4e33a6a.png)

Reply:
Added